---
title: "Exce&#231;&#245;es de solu&#231;&#227;o de problemas: System.Workflow.Runtime.WorkflowOwnershipException | Microsoft Docs"
ms.custom: ""
ms.date: "10/29/2016"
ms.prod: "visual-studio-dev14"
ms.reviewer: ""
ms.suite: ""
ms.technology: 
  - "devlang-csharp"
ms.tgt_pltfrm: ""
ms.topic: "article"
f1_keywords: 
  - "EHWRWorkflowOwnership"
helpviewer_keywords: 
  - "Exceção System.Workflow.Runtime.WorkflowOwnershipException"
  - "Exceção WorkflowOwnershipException"
ms.assetid: 20291283-dfc8-4e34-b84d-f380e04be376
caps.latest.revision: 7
caps.handback.revision: 7
author: "mikeblome"
ms.author: "mblome"
manager: "douge"
---
# Exce&#231;&#245;es de solu&#231;&#227;o de problemas: System.Workflow.Runtime.WorkflowOwnershipException
Um <xref:System.Workflow.Runtime.WorkflowOwnershipException> exceção é lançada quando o mecanismo de tempo de execução do fluxo de trabalho tenta carregar uma instância de fluxo de trabalho que é carregada no momento por outra instância de mecanismo de tempo de execução de fluxo de trabalho. Além disso, essa exceção é lançada quando o mecanismo de tempo de execução do fluxo de trabalho tenta salvar um fluxo de trabalho após o tempo limite da propriedade especificado enquanto o fluxo de trabalho estava carregando expirou.  
  
## Consulte também  
 <xref:System.Workflow.Runtime.WorkflowOwnershipException>   
 [Use the Exception Assistant](../Topic/How%20to:%20Use%20the%20Exception%20Assistant.md)